caravel/assets/javascripts/explorev2/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ExploreViewContainer from './components/ExploreViewContainer';
+
+const exploreViewContainer = document.getElementById('js-explore-view-container');
+const bootstrapData = exploreViewContainer.getAttribute('data-bootstrap');
```

Comment:
@vera-liu this is what i was thinking for bootstrapping the data from the server to the client.

we need to fix the formatting of `bootstrapData` either in js or python since this is not quite well formed json yet, but it gives you the idea.

<img width="1280" alt="screenshot 2016-09-13 17 59 54" src="https://cloud.githubusercontent.com/assets/130878/18496632/40477394-79dc-11e6-9ac0-02c7e67912f7.png">